.circleci/config.yml

```diff
@@ -0,0 +1,134 @@
+---
+version: 2
+
+jobs:
+  test:
+    docker:
+    - image: circleci/golang:1.10
+    working_directory: /go/src/github.com/prometheus/prometheus
+    resource_class: large
+
+    steps:
+    - checkout
+    - run: make promu
+    - run: make check_license style unused staticcheck build
```

Comment:
is it worth  contacting the circle support about the OOM when running `make test` ? It would be nice if we didn't need travis anymore.

Reply:
I did, I asked about upgrading to the larger containers, but they say we have to pay for it.  I'm working out if that's necessary for our project.  If we need to, we can get the CNCF to pay for it.

Reply:
In that case  I think just leave it for now and lets see what happens with the new CI we are working on.